library/ctr_drbg.c

```diff
@@ -66,6 +66,18 @@ void mbedtls_ctr_drbg_init( mbedtls_ctr_drbg_context *ctx )
  * Non-public function wrapped by mbedtls_ctr_drbg_seed(). Necessary to allow
```

Comment:
Commit "ctr_drbg: add comments relating the code with the NIST specification" doesn't (and other commits that were cherry-picked don't) need to say "cherry-picked from ...", especially not from a commit that was never part of a public long-lived branch. The author information is still showing @gilles-peskine-arm as author, so why mention the cherry pick?